e2e/app/xpath-builders.ts

```diff
@@ -62,7 +62,7 @@ export function buildXPath(xOpts: XPathOptions, container?: Container): string {
     selector = `${textExpr}//${type}`;
     break;
   case ElementType.Dropdown:
-    selector = `${textExpr}//*[contains(concat(" ", normalize-space(@class))," p-dropdown")]`;
+    selector = `${textExpr}${nodeLevel}//*[contains(concat(" ", normalize-space(@class), " "), " p-dropdown ")]`;
```

Comment:
Fix a bug in `selector` when only `nodeLevel` was specified. 
e.g. `const selectMenu = await SelectMenu.findByName(this.page, {ancestorLevel: 0}, this);`